app/containers/WriteReviewPage/index.js

```diff
@@ -12,8 +12,8 @@ import { compose } from 'redux';
 import injectReducer from 'utils/injectReducer';
 import injectSaga from 'utils/injectSaga';
 
-import { locationList, guideList, selectedLocationChange } from './actions';
-import { selectLocationList, selectGuideList, selectSelectedLocationId } from './selectors';
+import * as actions from './actions';
+import * as selectors from './selectors';
```

Comment:
Nice!!! Much cleaner

An alternative to this would be exporting a single object from each of those like 

```export default const selectors = {
   name1: selector1,
  ...
```
but this is cool too!

Reply:
Something that was really bothering me was rewriting function names everywhere, so typing up an object to export at the end of each file seemed just as redundant (it also makes files longer and adds another place to forget or mistype a new function (which is my main concern)).

I especially like this pattern (which I know would still work with the object) because it adds having to use "actions." or "selectors." so that it's explicit what the functions are and what file they are coming from. (I also read an article/blog that said it's easier to test functions set up with this pattern.)